assembler.ts

```diff
@@ -743,11 +736,13 @@ function resolveLabelsPass(
     behavior,
     resolveBp,
     resolveSub,
+    markParamRw,
   }: {
     labelInfo: LabelInfo;
     behavior: RawBehavior;
     resolveBp: (s: string) => RawBlueprint | undefined;
     resolveSub: (s: string) => ResolvedSub | undefined;
+    markParamRw: (s: number) => void;
```

Comment:
Can't this just write to `behavior.parameters` instead of using the callback?

Reply:
![](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExbGRkNW9uNzJoMjN1ZHNrNDZycHEwOGhoeHEwYWR3aXJzM2N6bHlhNCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/TJawtKM6OCKkvwCIqX/giphy.gif)